src/eventgrid/azext_eventgrid/_params.py

```diff
@@ -200,14 +218,22 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements
         c.argument('private_endpoint_connection_name', arg_type=private_endpoint_connection_name_type, options_list=['--name', '-n'], id_part='privateendpointconnections')
 
     with self.argument_context('eventgrid topic private-endpoint-connection list') as c:
-        c.argument('topic_name', arg_type=topic_name_type, options_list=['--name', '-n'], id_part=None)
+        c.argument('topic_name', arg_type=topic_name_type, id_part=None)
+
+    with self.argument_context('eventgrid topic private-endpoint-connection approve') as c:
+        c.argument('topic_name', arg_type=topic_name_type, id_part=None)
```

Comment:
Do we support connection id to specify a private end point connection?

Reply:
No (at least not currently).

Reply:
Because there  is no functionality difference with connection-id support, I will recommend you to add this support. In CLI other modules, for example, storage, keyvault, mysql, cosmosdb..., we all have the same interface as document shown.

Reply:
OK. thanks for the recommendation. We will not fix this as part of this PR and will evaluate it for future changes. Please feel free to open separate issue for that and Azure event grid service team will evaluate if we need to fix this and when. This should not block this PR. thx